### PR TITLE
Teku testnet

### PR DESCRIPTION
**Pull request by BeroBurny**
required for #528

### CODE REVIEW

File: teku-testnet.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: '3.6'
+
+####
+# https://github.com/ConsenSys/teku/tree/master/test-network
+
+services:
+  teku:
+    image: "${DOCKER_TEKU_IMAGE}"
```
Comment (mpetrunic) [inline teku-testnet.yml:8]: add default value?

File: testnet/teku/start.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ --network=minimal \
+ --rest-api-enabled \
+ --rest-api-cors-origins=http://localhost:2003 \
+ --rest-api-host-allowlist=* \
+ --log-file=/data/teku.log \
+ --data-path=/data \
+ --data-storage-mode=archive \
+ --Xinterop-genesis-time=${GENESIS_TIME} \
+ --Xinterop-owned-validator-start-index=0 \
+ --Xinterop-owned-validator-count=8 \
```
Comment (mpetrunic) [inline testnet/teku/start.sh:17]: should be one less than total to allow for chainguardian validator

File: teku-testnet.yml (hunk 1 above)
Comment (BeroBurny) [reply teku-testnet.yml:8]: forget to add on `.env.dist`

Comment (mpetrunic) [inline teku-testnet.yml:8]:
```suggestion
    image: "${DOCKER_TEKU_IMAGE:-consensys/teku:21.2}"
```

Comment (BeroBurny) [reply teku-testnet.yml:8]: I want to avoid this, as we use a centralized env file
we want to have a lower surface of refactoring when some update apear

Comment (mpetrunic) [reply teku-testnet.yml:8]: env file will overide this anyways

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**